README.md

```diff
@@ -303,66 +294,60 @@ Currently it supports multiple image selection and translations, and WIP we are
 
+* **ToggleOrder (<kbd>j</kbd>, <kbd>k</kbd>)**
 
-- **ToggleOrder (<kbd>j</kbd>, <kbd>k</kbd>)**
+  * If you have multiple images, use this to switch an individual image's overlap back and forth into view. Employs [`bringToFront()`](https://leafletjs.com/reference-1.5.0.html\#imageoverlay-bringtofront) and [`bringToBack()`](https://leafletjs.com/reference-1.5.0.html#imageoverlay-bringtoback) from the Leaflet API.
 
-  - If you have multiple images, use this to switch an individual image's overlap back and forth into view. Employs [`bringToFront()`](https://leafletjs.com/reference-1.5.0.html\#imageoverlay-bringtofront) and [`bringToBack()`](https://leafletjs.com/reference-1.5.0.html#imageoverlay-bringtoback) from the Leaflet API.
+* **ToggleOutline (<kbd>o</kbd>)**
+* **ToggleTransparency (<kbd>t</kbd>)**
+* **Revert**
 
-- **ToggleOutline (<kbd>o</kbd>)**
+  * Restores the image to its original proportions and scale, but keeps its current rotation angle and location on the map intact.
 
-- **ToggleTransparency (<kbd>t</kbd>)**
+* **Export**
+* **Delete (<kbd>delete</kbd>, <kbd>backscpace</kbd>)**
 
-- **Revert**
-
-  - Restores the image to its original proportions and scale, but keeps its current rotation angle and location on the map intact.
-
-- **Export**
-
-- **Delete (<kbd>delete</kbd>, <kbd>backscpace</kbd>)**
-
-  - Permanently deletes the image from the map.
+  * Permanently deletes the image from the map.
 
  Addons:
 
-- **ToggleRotate** (<kbd>caps lock</kbd>):
-
-  - Toggles between [rotate mode](#rotate) and [distort mode](#distort-(_default_)).
-  - Replaced as a default toolbar action by `ToggleRotateScale`, but still accessible via its hotkey, `mode`, and (WIP) custom toolbar actions API.
+* **ToggleRotate** (<kbd>caps lock</kbd>):
 
+  * Toggles between [rotate mode](#rotate) and [distort mode](#distort-(_default_)).
+  * Replaced as a default toolbar action by `ToggleRotateScale`, but still accessible via its hotkey, `mode`, and (WIP) custom toolbar actions API.
 
-- **ToggleScale** (<kbd>s</kbd>):
+* **ToggleScale** (<kbd>s</kbd>):
 
-  - Toggles between [scale mode](#scale) and [distort mode](#distort-(_default_)).
-  - Replaced as a default toolbar action by `ToggleRotateScale`, but still accessible via its hotkey, `mode`, and (WIP) custom toolbar actions API.
-
--   **EnableEXIF (WIP)**
+  * Toggles between [scale mode](#scale) and [distort mode](#distort-(_default_)).
+  * Replaced as a default toolbar action by `ToggleRotateScale`, but still accessible via its hotkey, `mode`, and (WIP) custom toolbar actions API.
 
+* **EnableEXIF (WIP)**
 
 <hr>
 
-### Multiple Image Interface:
+### Multiple Image Interface
 
 <hr>
 
 Defaults:
 
--   **Exports**
+* **Exports**
 
--   **Deletes (<kbd>delete</kbd>, <kbd>backscpace</kbd>)**
+* **Deletes (<kbd>delete</kbd>, <kbd>backspace</kbd>)**
 
-    - Permanently deletes groups of selected images from the map.
+  * Permanently deletes groups of selected images from the map.
```

Comment:
Should we note it has a `confirm()` just in case?

Reply:
yes will do